RUFAS/routines/manure/manure_nutrients/manure_nutrients.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import annotations
+
+from dataclasses import dataclass, fields
+
+
+@dataclass(kw_only=True, frozen=True)
+class ManureNutrients:
+    """A class to store the relevant manure nutrient information to be passed to the crop and soil module"""
+
+    nitrogen: float = 0.0
+    """Amount of accumulated manure nitrogen derived from the manure module, kg."""
```

Comment:
```suggestion
    """Amount of accumulated manure nitrogen derived from the manure module (kg)"""
```
Not an important issue at all, but in the Crop and Soil module we always format the units like this. Doesn't need to changed here  (especially if this is the style the Manure module already uses), but it's another thing we should decide a team standard for at some point.

Reply:
Great that you pointed this out. I could never be consistent about this. We have several choices:
- `description, <unit>.`
- `description, in <unit>.`
- `description (<unit>).`
- `description, measured in <unit>.`

I tend to flip-flop between the first two. But I can adopt your convention too. It looks good. 

Another thing is when there are no units, I think `unitless` sounds better, but technically `dimensionless` is more correct.

Reply:
I think the third option, `description, (<unit>).`,  would be best because it makes the description a proper sentence which will benefit the sphinx documentation greatly (IMO).

Also I agree that `unitless` sounds better, but maybe it would be best to let the SME's decide if it is important enough to use `dimensionless` in the docstrings.